language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -565,6 +567,52 @@ writePackageIdTs dir pkgId =
     T.writeFileUtf8 (dir </> "packageId.ts") $ T.unlines
       ["export default '" <> unPackageId pkgId <> "';"]
 
+writeIndexTs :: FilePath -> [Module] -> IO ()
+writeIndexTs packageSrcDir modules =
+  T.writeFileUtf8 (packageSrcDir </> "index.ts") lines
+  where
+    lines :: T.Text
```

Comment:
This looks confusing. If you want to call this `lines` it should be of type `[T.Text]` and the call site should call `T.unlines`. Otherwise just call it `content` or something like this.

Reply:
ok